bin/write-mappings.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
+
+// This executable can get more sophisticated in the future.
+// Possible improvements are:
+//   * Using reflection to not need a manual mapping_files Array
+//   * Taking an outputdirectory flag
+
+const fs = require('fs')
+const path = require('path')
+
+let mapping_files = [
+  'by_patron_type_factory',
+  'by_recap_customer_code_factory',
+  'by_sierra_location_factory'
+]
+
+let filesWritten = []
+
+mapping_files.forEach((mapping_file) => {
+  let mapping = require(path.join(__dirname, '..', 'lib', mapping_file)).createMapping()
+  let output_file_name = `${mapping_file.replace('_factory', '')}.json`
+
```

Comment:
Doh! I meant for output to be in sourcecontrol except for .json files.
I think that means amending .gitignore to have.

```
/output/*
!/output/.gitkeep
```
Which is what rails does for its ./log directory.
Which do you prefer?